File: backend/opencourse_api.js
const axios = require('axios')

export const getDeAnzaDept = async (dept) => {
  const url = `https://opencourse.dev/da/depts/${dept}/courses`
  const { data } = await axios.get(url)
  return data
}

// getDeAnzaDept('MATH')

// export const GET_COURSES = () => {
//   let response = new Array()
//   fetch('https://opencourse.dev/da/depts').then((r) => r.json())
//   console.log('r', response)

//   return response
// }

// export const getCourses = fetch('https://opencourse.dev/da/depts')
//   .then((r) => r.json())
//   .then(function (r) {
//     return r
//   })

File: backend/opencourseapi.py
import requests


class opencourse_api:
    def __init__(self, campus):
        self._API_URL = 'https://opencourse.dev'
        self._campus = campus

    def get_(self):
        req = requests.get(f'{self._API_URL}/{self._campus}')
        if req.ok:
            courses = req.json()

            for course in courses:
                print(course['dept'], course['course'], course['title'])
